packages/slate-editor/src/extensions/web-bookmark/components/WebBookmarkMenu.tsx

```diff
@@ -44,16 +44,19 @@ const LAYOUT_OPTIONS: OptionsGroupOption<BookmarkNode.Layout>[] = [
     },
 ];
 
-const PRESENTATION_OPTIONS: OptionsGroupOption<Presentation>[] = [
-    {
-        value: 'embed',
-        label: 'Embed',
-    },
-    {
-        value: 'card',
-        label: 'Bookmark',
-    },
-];
+function getPresentationOptions(element: BookmarkNode) {
+    return [
+        {
+            value: 'embed',
+            label: 'Embed',
+            disabled: !element.oembed.html,
+        },
+        {
+            value: 'card',
+            label: 'Bookmark',
+        },
+    ] as OptionsGroupOption<Presentation>[];
+}
```

Comment:
I usually avoid `as` casting, when possible.

```suggestion
function getPresentationOptions(element: BookmarkNode): OptionsGroupOption<Presentation>[] {
    return [
        {
            value: 'embed',
            label: 'Embed',
            disabled: !element.oembed.html,
        },
        {
            value: 'card',
            label: 'Bookmark',
        },
    ];
}
```